vnext/Desktop.IntegrationTests/React.Windows.Desktop.IntegrationTests.vcxproj

```diff
@@ -61,6 +61,7 @@
       <AdditionalIncludeDirectories>$(VCInstallDir)UnitTest\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
       <UseFullPaths>true</UseFullPaths>
+      <LanguageStandard>stdcpp17</LanguageStandard>
```

Comment:
[nit] Not needed.

Reply:
Removed.

---
In reply to: [296965215](https://github.com/microsoft/react-native-windows/pull/2656#discussion_r296965215) [](ancestors = 296965215)